system_tests/ec2_handler.py

```diff
@@ -359,12 +363,20 @@ def _elbs(self, elb_client):
                 for elb in elb_client.get_all_load_balancers()]
 
     def _vpcs(self, vpc_client):
-        return [(vpc.id, vpc.id)
-                for vpc in vpc_client.get_all_vpcs()]
+        vpcs = []
+        for vpc in vpc_client.get_all_vpcs():
+            if vpc.is_default:
+                continue
+            vpcs.append(vpc.id)
+        return vpcs
 
     def _subnets(self, vpc_client):
-        return [(subnet.id, subnet.id)
```

Comment:
[(subnet.id, subnet.id) for subnet in vpc_client.get_all_subnets() if not subnet.defaultForAz]